src/sgame/sg_main.cpp

```diff
@@ -2273,6 +2276,10 @@ can see the last frag.
 */
 void CheckExitRules()
 {
```

Comment:
It should work, see https://github.com/DaemonEngine/Daemon/blob/master/src/shared/CommonProxies.cpp that actually implements vmCvar_t on top of Cvar::Cvar